pkg/controller/operators/catalog/operator.go

```diff
@@ -2204,6 +2202,10 @@ func validateV1Beta1CRDCompatibility(dynamicClient dynamic.Interface, oldCRD *ap
 	return validateExistingCRs(dynamicClient, gr, validationsMap)
 }
 
+type ValidationError struct {
+	error
+}
```

Comment:
Can this be unexported?

Reply:
Sure

Reply:
Done in https://github.com/operator-framework/operator-lifecycle-manager/pull/3209/commits/81d1ed44802c5218563f17a15c7d4665db578f84